persistence-model/src/test/scala/hmda/persistence/serialization/parser/HmdaFileParserProtobufConverterSpec.scala

```diff
@@ -0,0 +1,58 @@
+package hmda.persistence.serialization.parser
+
+import hmda.model.fi.lar.LarGenerators
+import hmda.model.fi.ts.TsGenerators
+import hmda.model.parser.LarParsingError
+import hmda.persistence.messages.events.processing.HmdaFileParserEvents.{ LarParsed, LarParsedErrors, TsParsed, TsParsedErrors }
+import hmda.persistence.model.serialization.HmdaFileParserEvents._
+import hmda.persistence.serialization.parser.HmdaFileParserProtobufConverter._
+import org.scalacheck.Gen
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class HmdaFileParserProtobufConverterSpec extends PropSpec with PropertyChecks with MustMatchers
+    with TsGenerators with LarGenerators {
+
+  def listOfStringsGen: Gen[List[String]] = Gen.listOf(Gen.alphaStr)
+  def intGen: Gen[Int] = Gen.choose(0, 100000)
```

Comment:
For me, these types greatly improve the readability of the code. Also, see my comment below.